alacritty_terminal/src/term/mod.rs

```diff
@@ -2230,6 +2242,16 @@ impl<T: EventListener> Handler for Term<T> {
             self.set_title(popped);
         }
     }
+
+    #[inline]
+    fn window_size_pixels<W: io::Write>(&mut self, writer: &mut W) {
+        let _ = write!(writer, "\x1b[4;{};{}t", self.window_height, self.window_width);
+    }
```

Comment:
This reports the inner window size, not the text area size.

Reply:
What should be sent? `width + padding_x`?

Reply:
You'd have to subtract the padding from the inner size.

Reply:
Is this correct?

```rust
self.window_height = (size.height - size.padding_y * 2) as usize;
self.window_width = (size.width - size.padding_x * 2) as usize;
```